programs/clearing_house/src/state/events.rs

```diff
@@ -147,6 +147,29 @@ impl Default for OrderAction {
     }
 }
 
+#[event]
+#[derive(Default)]
+pub struct LPRecord {
```

Comment:
i dont think this is a sufficient amount of logging. we want to be able understand what the users performance was. you should sync with z for all the data we'll want